tools/stove-cli/spa/src/index.css

```
@@ -80,10 +94,2611 @@
+  border: 1px solid var(--stove-border);
+  border-radius: 999px;
+  background: color-mix(in srgb, var(--stove-surface) 70%, transparent);
+  padding: 6px 10px;
+  white-space: nowrap;
+}
+
+.stove-stream-status > span {
+  width: 6px;
+  height: 6px;
+  border-radius: 999px;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)